RELEASE_HOWTO.md

```diff
@@ -0,0 +1,14 @@
+# Steps for doing a xdg-desktop-portal release
+
+ - git clean -fxd
+ - bump version in configure.ac
+ - add content to NEWS
+ - git commit -m <version>
+ - git push origin master
+ - ./autogen.sh --enable-docbook-docs 
+ - make all
+ - make distcheck
+ - git tag <version>
+ - git push origin refs/tags/<version>
```

Comment:
I think I'd do `git push <version>`, which should work fine too (unless there's a branch with the same name).

But another option (and this is how I make releases on Endless) that I kind of like a bit more is **atomically** pushing the branch and the tags at the same time: don't do the `git push origin master` before, and simply do here `git push origin master <version>`